iogt/static/css/variables/_colors.scss

```diff
@@ -7,13 +7,21 @@ $roman          : #df6859;
 $danube         : #6093cd;
 $dove-gray      : #737373;
 
+$pelorous       : #37BFBE;
+$eucalyptus     : #2A9B58;
+$saffron        : #F2B438;
+$sandy-brown    : #EF9955;
+$cinnabar       : #EC3B3A;
+$medium-violet  : #B62A99;
+$indigo_blue    : #5F7AC9;
+
 $color_list: (
-  de-york        : $de-york,
-  celery         : $celery,
-  mandy          : $mandy,
-  wewak          : $wewak,
-  robin-egg-blue : $robin-egg-blue,
-  roman          : $roman,
-  danube         : $danube,
-  dove-gray      : $dove-gray
+  pelorous        : $pelorous,
+  eucalyptus      : $eucalyptus,
+  saffron         : $saffron,
+  sandy_brown     : $sandy-brown,
```

Comment:
why are we sometimes using an underscore `_` and other times a dash `-`? Why did you rename the colours?